mmv1/products/notebooks/api.yaml

```diff
@@ -281,6 +281,37 @@ objects:
               Defines whether the instance has the vTPM enabled.
               Enabled by default.
             default_value: true
+      - !ruby/object:Api::Type::Enum
+        name: 'nicType'
+        description: |
+          The type of vNIC driver.
+        values:
+          - UNSPECIFIED_NIC_TYPE
+          - VIRTIO_NET
+          - GVNIC         
+      - !ruby/object:Api::Type::NestedObject
+        name: 'reservationAffinity'
+        description: |
+          Reservation Affinity for consuming Zonal reservation.
+        properties:
+          - !ruby/object:Api::Type::Enum
```

Comment:
I can't comment directly on L298 for some reason. Weird. Are you sure the field is called `type`? While the type of the field is called `Type`, the field itself is called `consume_reservation_type` in the RPC docs: https://cloud.google.com/notebooks/docs/reference/rpc/google.cloud.notebooks.v1#google.cloud.notebooks.v1.ReservationAffinity

Reply:
Good catch, changing this now.
Github has been having all sorts of issues since yesterday.